src/predict_request_validation_utils.cpp

```diff
@@ -828,11 +830,20 @@ Status RequestValidator<ovms::InferenceRequest, InferenceTensor, const Inference
         SPDLOG_DEBUG(details);
         return Status(StatusCode::INVALID_CONTENT_SIZE, details);
     }
+    if (expectedPrecision == Precision::STRING) {
+        SPDLOG_DEBUG("Skipping check for content size due to string precision");
+        return StatusCode::OK;
```

Comment:
Here you omit not only content size check, but all validation steps below.
Not only it is unsafe because we may add further steps below and those will be skipped, but we **already** do:
```
return validateInferenceTensorBufferType(tensor);
``` 
line 855